src/stories/Icon.stories.tsx

```diff
@@ -0,0 +1,171 @@
+import { Flex, H3 } from 'honorable'
+
+import IconFrame from '../components/IconFrame'
+
+export default {
+  title: 'Icon',
```

Comment:
Change this to 'Icon Frame' and I'm good. :)

Reply:
Oh man... This is not my day :slightly_smiling_face: